src/main/webapp/WEB-INF/tags/instructor/studentList.tag

```diff
@@ -134,7 +134,7 @@
                       data-placement="top">
                     Edit
                   </a>
-                  <c:if test="${fromCourseDetailsPage && student.studentStatus == STUDENT_COURSE_STATUS_YET_TO_JOIN}">
+                  <c:if test="${(fromCourseDetailsPage || fromStudentListPage) && student.studentStatus == STUDENT_COURSE_STATUS_YET_TO_JOIN}">
```

Comment:
This fix, whilst fixes the problem, introduces another problem. Now when instructors click on "send invite" in the Students page, the website redirects to the Courses page(and without the confirmation message that sending invite in Courses page would trigger). Can you attempt to solve the issue inside the Students page itself?

Reply:
Yes sure! I am on it.

Reply:
@Haozhe321 I am new to the contributors section here, so apologies for dumb questions. I have fixed the issue with redirection. However, I am having trouble showing the confirmation message. From what I've understood, the code that provides this functionality exists in instructorCourseDetails.js. Can you help me understand if this code is generated, or written?

Reply:
@Haozhe321 Apologies for the somewhat useless commits. I am a new contributor so took some time getting used to the tests. 

I added the 2 changes you asked for. 
For fixing the redirection, I created a separated Action class for invite button in instructorStudentList. The code is very similar to the one for instructorCourseDetailsAction. 
For fixing the confirmation message, I moved the corresponding code in instructorCourseDetails.js to instructor.js ("AttachEventToSendInviteLink" - in order to avoid redundancy when using it with instructorStudentList), imported it and called it accordingly.

Can you please have a look and give me feedback? It works as far as I can tell. Thank you.